backends/tofino/bf-p4c/arch/v1model.cpp

```diff
@@ -2471,8 +2470,8 @@ bool AddAdjustByteCount::preorder(IR::Declaration_Instance *decl) {
                     new IR::Vector<IR::Type>(
                         {new IR::Type_Name("__bfp4c_bridged_metadata_header")}),
                     new IR::Vector<IR::Argument>({new IR::Argument(new_abc_member)}));
-                decl->annotations =
-                    decl->annotations->addAnnotation("adjust_byte_count"_cs, new_abc_expr);
+                IR::Annotations::addOrReplace(decl->annotations, "adjust_byte_count"_cs,
```

Comment:
Here we could also use decl->addOrReplace, no?

Reply:
Yeah, one thing I realized while working on this is that we now have the IAnnotated interface and the Annotation helpers which have some redundancy.

Reply:
Yes, the difference is the node itself that carries annotations and just vector of annotations that might be changed standalone w/o node (e.g. before creation).